### DISCUSSION TITLE
[安全提醒]该帐号因使用非官方QQ客户端被暂时  冻结，请点击下方按钮了解详情及处理。(错误码：40)

### DISCUSSION THREAD

**Discussion by chinoll** [takayama-lily/oicq#243] [Q&A]
[安全提醒]该帐号因使用非官方QQ客户端被暂时 冻结，请点击下方按钮了解详情及处理。(错误码：40)
这个如何解决

**Comment by 468336329Zc** [comment]
换一个账号 兄弟 你的nodejs用的什么版本 我出现 连接可用服务器失败  是什么错误  就在问题中的。

**Comment by takayama-lily** [comment]
暂时不清楚原因，小概率发生，可以看做是误封。
mamoe/mirai/issues/1461

**Comment by Cloud370** [comment]
一般这样的话 是账号权重不够 或者触发风控了 建议你先用自己的设备登录几天 再用扫码登录oicq 然后一周之后稳定了 再用密码登录 我5 6个bot在线 有小号有大号 除非直接异地登录 不然不会出现风控的问题

**Reply by Baytars** [reply]
用大号登正常工作，用小号登，除了手机显示登录成功外，控制台按回车提示
```
[ERROR] [Android:Q号] - [登录失败]请你稍后重试。(0x30)
```
登不上去，是不是也是这个原因？
